test/parallel_api/ranges/sort_ranges_sycl.pass.cpp

```diff
@@ -53,6 +53,25 @@ main()
 
     bool res2 = ::std::is_sorted(data2, data2 + max_n, ::std::greater<int>());
     EXPECT_TRUE(res2, "wrong effect from 'sort with comparator' with sycl ranges");
+
+   //test with random number and projection usage
+   std::default_random_engine gen{std::random_device{}()};
+   std::uniform_real_distribution<float> dist(0.0, 100.0);
+
+   const int N = 1 << 20;
```

Comment:
I would prefer `constexpr std::size_t N = 1 << 20` and do forget to include `<cstddef>` if you follow the suggestion